packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.test.js

```diff
@@ -63,24 +61,24 @@ describe('<SpanBarRow>', () => {
   beforeEach(() => {
     props.onDetailToggled.mockReset();
     props.onChildrenToggled.mockReset();
-    wrapper = mount(<SpanBarRow {...props} />);
+    render(<SpanBarRow {...props} />);
```

Comment:
i think we can move this render call out of the `beforeEach` and put it in each individual test.

Reply:
also the `let wrapper;` declaration on line 59 isn't needed anymore